bin/configs/spring-boot-source.yaml

```diff
@@ -0,0 +1,10 @@
+generatorName: spring
+outputDir: samples/openapi3/server/petstore/springboot-source
```

Comment:
can you please add the new sample folder to https://github.com/OpenAPITools/openapi-generator/blob/master/.github/workflows/samples-spring.yaml#L18-L42 so that the CI (github actions) will test it?

Reply:
I think I have done it with https://github.com/OpenAPITools/openapi-generator/pull/11531/commits/87367b6bddd8efb096a8d54f59970da43b8d36aa.

Please review.